VER-1001 Additional post-integration fixes for vertex_fact_loan_purchase

rschoenbeck opened the pull request:
Small tweaks for post-integration changes

vanmh commented:
It would be good to add a test to confirm that direct loan purchases are being included.  It could be something like a point in time count or sum.

rschoenbeck commented:
Added test for direct loan purchase count, modified the one for autolending to make sure we're specifically picking up Direct autolending as well, and also tweaked a count check based on a comment.

vanmh commented:
Looks good.
